apiserver/v1/apiserver.openshift.io_apirequestcount.yaml

```diff
@@ -0,0 +1,296 @@
+  name: apirequestcounts.apiserver.openshift.io
+spec:
+  group: apiserver.openshift.io
+  names:
+    kind: APIRequestCount
+    listKind: APIRequestCountList
+    plural: apirequestcounts
+    singular: apirequestcount
+  scope: Cluster
+  versions:
+  - name: v1
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    "schema":
+      "openAPIV3Schema":
+        description: APIRequestCount tracts requests made to a deprecated API. The
+          instance name should be of the form `resource.version.group`, matching the
+          deprecated resource.
+        type: object
+        required:
+        - spec
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: spec defines the characteristics of the resource.
+            type: object
+            properties:
+              numberOfUsersToReport:
+                description: numberOfUsersToReport is the number of users to include
+                  in the report. If unspecified or zero, the default is ten.  This
+                  is default is subject to change.
+                type: integer
+                format: int64
+                maximum: 100
+                minimum: 0
+          status:
+            description: status contains the observed state of the resource.
+            type: object
+            properties:
+              conditions:
+                description: conditions contains details of the current status of
+                  this API Resource.
+                type: array
+                items:
+                  description: "Condition contains details for one aspect of the current
+                    state of this API Resource. --- This struct is intended for direct
+                    use as an array at the field path .status.conditions.  For example,
+                    type FooStatus struct{     // Represents the observations of a
+                    foo's current state.     // Known .status.conditions.type are:
+                    \"Available\", \"Progressing\", and \"Degraded\"     // +patchMergeKey=type
+                    \    // +patchStrategy=merge     // +listType=map     // +listMapKey=type
```

Comment:
@sttts @deads2k i really wonder why the generator comments are copied to description